components/python/jinja2/Makefile

```diff
@@ -11,11 +11,12 @@
 #
 # Copyright (c) 2014 Alexander Pyhalov.  All rights reserved.
 # Copyright 2015, PALO Richard.
+# Copyright 2017 Gary Mills
 #
 
 include ../../../make-rules/shared-macros.mk
 
-PATH=/usr/bin:/usr/gnu/bin:/usr/sbin
+PATH = $(PATH.ill):/usr/sbin
```

Comment:
I examined 101 component Makefiles.  11 of them contained /usr/sbin in the PATH, and six of them contained /usr/perl5/bin .  There were no others.  I can append /usr/sbin:/usr/perl5/bin to the two PATH symbols in shared-macros.mk .  I'll also change the name of PATH.ill to PATH.illumos in that same file.  Fixing that will be one commit.  The second commit will update a batch of component Makefiles.